common/src/state/action.rs

```diff
@@ -135,25 +127,19 @@ pub enum Action {
     RemoveFromSidebar(Uuid),
     /// Adds or removes a chat from the favorites page
     #[display(fmt = "ToggleFavorite")]
-    ToggleFavorite(Chat),
+    ToggleFavorite(&'a Uuid),
```

Comment:
`Uuid` implements `Copy` so you could easily dereference it without needing to borrow it